video/out/win32/droptarget.c

```diff
@@ -156,8 +156,10 @@ static STDMETHODIMP DropTarget_Drop(IDropTarget *self, IDataObject *pDataObj,
                 wchar_t *buf = talloc_array(NULL, wchar_t, len + 1);
 
                 if (DragQueryFileW(drop, i, buf, len + 1) == len) {
-                    char *fname = mp_to_utf8(files, buf);
+                    wchar_t *target = mp_w32_get_shell_link_target(buf);
```

Comment:
To be honest I don't like it living here. Wouldn't it be better to resolve it when the file is opened? if it is a common code, we can have `#ifdef` no biggie.

Reply:
Also using `.lnk` in 2023. But I guess this is still the option in context menu... Nowadays I always use symbolic links, they are more robust and don't require special handling in app.